chalicelib/checks/higlass_checks.py

```diff
@@ -0,0 +1,1241 @@
+            discovered files.
+        error(string, optional, default="") : Describes any errors generated.
+    """
+
+    # Collect all of the Processed files with a higlass uid.
+    processed_files = []
+
+    if "processed_files" in expset:
+        # The Experiment Set may have Processed Files.
+        processed_files = [ pf for pf in expset["processed_files"] if "higlass_uid" in pf ]
+
+    # Search each Experiment, they may have Processed Files.
+    if "experiments_in_set" in expset:
+        for experiment in [ exp for exp in expset["experiments_in_set"] if "processed_files" in exp]:
+            exp_processed_files = [ pf for pf in experiment["processed_files"] if "higlass_uid" in pf ]
+            processed_files += exp_processed_files
+
+    if len(processed_files) < 1:
+        return {
+            "error": "No processed files found",
+            "files": [],
+            "genome_assembly": "",
+        }
+
+    # Make sure all of them have the same genome assembly.
+    genome_assembly_set = { pf["genome_assembly"] for pf in processed_files if "genome_assembly" in pf }
+
+    if len(genome_assembly_set) > 1:
+        return {
+            "error": "Too many genome assemblies {gas}".format(gas=genome_assembly_set),
+            "files": [],
+            "genome_assembly": ""
+        }
+
+    # Return all of the processed files.
+    unique_accession = {}
+    unique_files = []
+    for pf in processed_files:
+        if pf["accession"] in unique_accession:
+            continue
+        unique_accession[pf["accession"]] = pf
+        unique_files.append(pf["accession"])
+
+    return {
+        "error": "",
+        "files": unique_files,
+        "genome_assembly": processed_files[0]["genome_assembly"]
+    }
+
+@check_function(expset_accession=None)
+def check_expsets_otherprocessedfiles_for_higlass_viewconf(connection, **kwargs):
+    """ Check to generate Higlass view configs on Fourfront for Experiment Sets Other Processed Files (aka Supplementary Files.)
+
+        Args:
+            connection: The connection to Fourfront.
+            **kwargs, which may include:
+                expset_accession: Only check this expset.
+
+        Returns:
+            check results object.
+    """
+
+    check = init_check_res(connection, 'check_expsets_otherprocessedfiles_for_higlass_viewconf')
+    check.full_output = {}
+    check.action = 'patch_expsets_otherprocessedfiles_for_higlass_viewconf'
+
+    # If an accession was specified, get it
+    if kwargs['expset_accession']:
+        expsets_by_accession = {
+            kwargs['expset_accession'] : ff_utils.get_metadata(kwargs['expset_accession'], key=connection.ff_keys, ff_env=connection.ff_env, add_on="frame=embedded")
+        }
+    else:
+        # Otherwise search for all relevant Experiment Sets
+        # get the fields you need to include
+        fields_to_include = "&field=" + "&field=".join(
```

Comment:
The first `&field=` is unnecessary. Doesn't seem to break anything but maybe remove